python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -460,6 +461,13 @@ def wait_for_pod(
                     self.logger('Waiting for pod "%s" to initialize...' % pod_name)
                     self.sleeper(wait_time_between_attempts)
                     continue
+                if state.waiting.reason == KubernetesWaitingReasons.CreateContainerConfigError:
```

Comment:
Can we also filter on the message so we only retry for this secrets issue? Otherwise, we'll have to wait for the end of our timeout for any container config error (not sure how common those are)